packages/graphql-transformer-core/src/GraphQLTransform.ts

```diff
@@ -291,7 +299,7 @@ export default class GraphQLTransform {
             }
             if (matchDirective(transformer.directive, dir, def)) {
                 if (isFunction(transformer.object)) {
-                    const transformKey = makeSeenTransformationKey(dir, def)
+                    const transformKey = makeSeenTransformationKey(dir, def, undefined, undefined, index)
```

Comment:
The key (that prevents running the same transformer at the same location twice) now includes an index because we enabled multiple directives of the same type to be put in the same location. This was previously validated against.